x/claim/keeper/keeper_test.go

```diff
@@ -67,6 +69,18 @@ func (suite *KeeperTestSuite) SetupTest() {
 	params := types.DefaultParams()
 	params.AirdropStartTime = suite.ctx.BlockTime()
 	suite.app.ClaimKeeper.SetParams(suite.ctx, params)
+
+	stakingParams := suite.app.StakingKeeper.GetParams(suite.ctx)
+	stakingParams.BondDenom = params.GetClaimDenom()
+	suite.app.StakingKeeper.SetParams(suite.ctx, stakingParams)
+
+	govParams := suite.app.GovKeeper.GetDepositParams(suite.ctx)
+	govParams.MinDeposit[0].Denom = params.GetClaimDenom()
```

Comment:
```suggestion
```

Reply:
we do need staking params though—TestDelegationAutoWithdrawAndDelegateMore otherwise will expect stake instead of evmos